develop/entities/effects.md

```diff
@@ -57,10 +58,30 @@ language file.
 }
 ```
 
-### Testing {#testing}
+### Applying The Effect {#applying-the-effect}
 
-Use the command `/effect give @p fabric-docs-reference:tater` to give the player our Tater effect.
-Use `/effect clear @p fabric-docs-reference:tater` to remove the effect.
+It's worth taking a look at how you'd typically apply an effect to an entity.
+
+::: tip
+For a quick test, it might be a better idea to use the previously mentioned `/effect` command: <br/>
+`/effect give @p fabric-docs-reference:tater`
```

Comment:
```suggestion
For a quick test, it might be a better idea to use the previously mentioned `/effect` command:

```mcfunction
/effect give @p fabric-docs-reference:tater
\```
```

^^ Without the backslash though...

Reply:
The backslash slipped through

Reply:
yeah I saw that